<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankReportWindow</class>
 <widget class="QMainWindow" name="BankReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание отчета для одного банка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="HeadingLabel">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Выберите параметры отчета</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="DescriptionTextBrowser">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Список возможных банков представлен в окне справа ниже.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Минимальная возможная дата: 01.01.2018.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Максимальная возможная дата: 31.12.2018.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="BankNameLabel">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Выберите название банка:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BankNameComboBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="StartDateLabel">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Выберите дату начала отчета:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="StartDateEdit">
        <property name="maximumDate">
         <date>
          <year>2018</year>
          <month>11</month>
          <day>30</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2018</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>MM.yyyy</string>
        </property>
        <property name="date">
         <date>
          <year>2018</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="EndDateLabel">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Выберите дату окончания отчета: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="EndDateEdit">
        <property name="maximumDate">
         <date>
          <year>2018</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2018</year>
          <month>2</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>MM.yyyy</string>
        </property>
        <property name="date">
         <date>
          <year>2018</year>
          <month>2</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="BackPushButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FormPushButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Сформировать отчет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
